portfolio/src/main/webapp/nickname.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
```

Comment:
Do we need the JS file for this page?

Reply:
I originally included a function but then I found that function should be in the servlet file. But I forgot to remove the JS file in the html file. I will remove this line in the newest commit.